File: bootstrap/v0.0.8.4.6/benchmarks/python/primes.py
def count_primes(limit):
    count = 0
    n = 2
    while n < limit:
        is_prime = 1
        i = 2
        while i * i <= n:
            remainder = n % i
            if remainder == 0:
                is_prime = 0
            i = i + 1
        if is_prime == 1:
            count = count + 1
        n = n + 1
    return count

if __name__ == "__main__":
    result = count_primes(10000)
    exit(result)

File: bootstrap/v0.0.8.4.5.2.4/benchmarks/rust/primes.rs
fn count_primes(limit: i32) -> i32 {
    let mut count = 0;
    let mut n = 2;
    while n < limit {
        let mut is_prime = 1;
        let mut i = 2;
        while i * i <= n {
            let remainder = n % i;
            if remainder == 0 {
                is_prime = 0;
            }
            i = i + 1;
        }
        if is_prime == 1 {
            count = count + 1;
        }
        n = n + 1;
    }
    count
}

fn main() {
    let result = count_primes(10000);
    std::process::exit(result);
}